docs/astro/src/content/docs/reference/std-widgets/basic-widgets/slider.mdx

```diff
@@ -80,7 +80,10 @@ Slider {
 
 ### orientation
 <SlintProperty propName="orientation" typeName="enum" enumName="Orientation" defaultValue="horizontal">
-If set to true the Slider is displayed vertical.
+```slint
+Slider {
+    orientation: Orientation.vertical; //valid
+}
```

Comment:
The code fence isn't closed - that will produce rendering errors.

I think it would be better to have a textual description than just a naked slint code snippet. And you're right, the boolean was wrong. How about this:

Describes the orientation of the Slider, vertical or horizontal.

Reply:
oops indeed.
Fixed.
Thanks